src/workflows/airqo_etl_utils/ml_utils.py

```
@@ -1029,6 +995,619 @@ def train_satellite_model(data):
+    def train_point_and_save_to_gcs(
+        df: pd.DataFrame,
+        *,
+        features: List[str],
+        target: str,  # "pm25_mean" or "pm25_min" or "pm25_max"
+        model_kind: str = "point",
+        date_col: str = "day",
+        test_fraction: float = 0.2,
+        random_state: int = 42,
+        lgb_params: Optional[Dict] = None,
+        # GCS
+        project_name: str,
```

Ruff (0.15.1)

[warning] Unused static method argument: `project_name`

(ARG004)